BGL/include/CGAL/boost/graph/helpers.h

```diff
@@ -737,7 +737,7 @@ bool is_degenerate_triangle_face(
 
 /**
  * \ingroup PkgBGLHelperFct
- * \brief Creates a triangulated regular prism
+ * \brief Creates a triangulated regular prism, cw oriented,
```

Comment:
what is cw oriented? Don't we have a function called `is_outward_oriented()`?

Reply:
cw is clockworck oriented.

Reply:
If you meant clockwise, It does not mean anything in 3D space. Use outward oriented.

Reply:
Yes I meant clockwise.